src/components/Footer.tsx

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import { Layout, Row, Col, Typography, Space, Button, Image } from "antd";
+import {
+  GithubOutlined,
+  TwitterOutlined,
+  MailOutlined,
+  LinkedinOutlined,
+} from "@ant-design/icons";
+
+const { Footer } = Layout;
+const { Text, Link } = Typography;
+
+const footerSections = [
+  {
+    title: "ABOUT",
+    links: [
+      { title: "About the AP", href: "https://accordproject.org/about/" },
+      {
+        title: "FAQ",
+        href: "https://accordproject.org/frequently-asked-questions/",
+      },
+    ],
+  },
+  {
+    title: "COMMUNITY",
+    links: [
+      { title: "Contribute", href: "https://accordproject.org/contribute/" },
+      {
+        title: "Tech WG",
+        href: "https://accordproject.org/working-groups/technology/",
+      },
+      {
+        title: "Join Discord",
+        href: "https://discord.com/invite/Zm99SKhhtA",
+        external: true,
+      },
+    ],
+  },
+  {
+    title: "PROJECTS",
+    links: [
+      {
+        title: "Concerto",
+        href: "https://accordproject.org/projects/concerto/",
+      },
+      {
+        title: "VS Code Extension",
+        href: "https://marketplace.visualstudio.com/items?itemName=accordproject.cicero-vscode-extension",
+      },
+      {
+        title: "Template Playground",
+        href: "https://playground.accordproject.org/",
+      },
+    ],
+  },
+  {
+    title: "RESOURCES",
+    links: [
+      {
+        title: "Template Library",
+        href: "https://templates.accordproject.org/",
+      },
+      { title: "Model Repository", href: "https://models.accordproject.org/" },
+      { title: "Videos", href: "https://vimeo.com/accordproject" },
+      {
+        title: "GitHub",
+        href: "https://github.com/accordproject",
+        external: true,
+      },
+      { title: "Documentation", href: "https://docs.accordproject.org/" },
+    ],
+  },
+];
+
+const CustomFooter: React.FC = () => {
+  const year = new Date().getFullYear();
+
+  return (
+    <Footer
+      style={{
+        background: "#1b2540",
+        color: "white",
+        padding: "80px 50px 20px 50px",
+      }}
+    >
+      <Row justify="space-between" align="top">
+        <Col span={7}>
+          <Space direction="vertical" size="middle">
+            <a
```

Comment:
Can you please use `<Link>` here as well? Let's be consistent with using ant-ui as much as we can.

Reply:
Sure. Would do